cfgov/data_research/management/commands/conference_export.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import print_function
+
+import csv
+import json
+import logging
+
+from cStringIO import StringIO
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+from django.utils.encoding import force_bytes
+from wagtail.wagtailcore.models import Page
+
+from data_research.models import ConferenceRegistration
+from v1.util.migrations import get_stream_data
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_registration_form_from_slug(slug):
+    page = Page.objects.get(slug=slug).specific
+    content = get_stream_data(page, 'content')
+
+    for block in content:
+        if 'conference_registration_form' == block['type']:
+            return block
+
+    raise RuntimeError('no registration form found on {}'.format(page))
+
+
+class ConferenceExporter(object):
+    FIELDS = (
+        ('name', 'Name'),
+        ('organization', 'Organization'),
+        ('email', 'Email Address'),
+        ('sessions', 'Selected Sessions'),
+        ('foodinfo', 'Food Restrictions'),
+        ('accommodations', 'Accommodations Needed'),
+        ('code', 'GovDelivery Code'),
+    )
+
+    EMAIL_SUBJECTS = (
+        '[2016 CFPB Research Conference] Attendance Update',
+        '[2016 CFPB Research Conference] Attendance Full',
+    )
+
+    EMAIL_BODIES = (
+        (
+            'The 2016 Research Conference form is at {} of {} capacity for '
+            'signups.'
+        ),
+        (
+            'The 2016 Research Conference form is at-capacity (full) and '
```

Comment:
Let's output both. I can foresee a scenario where the numbers get out of sync (because someone updates the Wagtail admin) and they get confused, so it's best to represent that here in the email at least...


```
The 2016 Research Conference form has reached capacity with {count} submissions. Signups are now closed. 

The capacity for this event was set to {capacity} at the time of this email.
```